src/main/java/com/gtnewhorizons/neid/NEIDConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.gtnewhorizons.neid;
+
+import java.io.File;
+
+import net.minecraft.launchwrapper.Launch;
+import net.minecraftforge.common.config.Configuration;
+
+public class NEIDConfig {
+
+    static Configuration config;
+    public static boolean catchUnregisteredBlocks;
+    public static boolean removeInvalidBlocks;
+    public static boolean postNeidWorldsSupport;
+    public static boolean extendDataWatcher;
+
+    public static void init(final File file) {
```

Comment:
why is this public static, is it used anywhere else ? can't you just make all of that a static block